clevis/Sanity/stderr/test.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   test.sh of /CoreOS/clevis/Sanity/stderr
+#   Description: Test that clevis displays eventual error messages when
+#                unlocking fails.
+#   Author: Sergio Correia <scorreia@redhat.com>
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Copyright (c) 2021 Red Hat, Inc.
+#
+#   This program is free software: you can redistribute it and/or
+#   modify it under the terms of the GNU General Public License as
+#   published by the Free Software Foundation, either version 2 of
+#   the License, or (at your option) any later version.
+#
+#   This program is distributed in the hope that it will be
+#   useful, but WITHOUT ANY WARRANTY; without even the implied
+#   warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#   PURPOSE.  See the GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public License
+#   along with this program. If not, see http://www.gnu.org/licenses/.
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# Include Beaker environment
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+PACKAGE="clevis"
+PACKAGES="${PACKAGE}-luks tang util-linux"
+
+PASSPHRASE=redhat123
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlAssertRpm --all
```

Comment:
feel free to remove the `rlDie` if you think it is a good idea to continue even with the missing requires.
```suggestion
        rlRun "rlCheckRecommended; rlCheckRequired" || rlDie 'cannot continue'
```